### ISSUE TITLE
LED Flash Brightness Issue 

### ISSUE THREAD

**Issue by jkykayex** [rzeldent/esp32cam-rtsp#47]
Caveat: I Could be mistaken due to being a total N00B. 

I ultimately intend to use this with HomeAssistant and integrate the LED brighness control over there, however, in my initial testing, I used chrome with the link in the instructions (http://192.168.x.xxx/flash?v=100 as the parameter (100) and the[ following test tool](https://reqbin.com/)

Both these methods returned the following responses: 

<img width="1344" alt="image" src="https://user-images.githubusercontent.com/67133996/226143754-cebe22bd-0dc0-4d95-8e0a-a3f27684e4bf.png">

**Comment by rzeldent** [comment]
Hi jkykayes,

I retested it with the current develop and at my side it is working as it should. I used RapidApi but this is yet another restclient.
The light turns on/off.

<img width="1022" alt="image" src="https://user-images.githubusercontent.com/7620231/226170893-d0e0bd7a-9bda-42de-bb59-24e26fbf3183.png">

Is it possible the password you are using is not correct. The password that you entered (yourself) for the AP password in the System configuration when configuring the WiFi....

![image](https://user-images.githubusercontent.com/7620231/226171790-78a14c2c-ec73-4eae-9682-63026ef225d3.png)

Could you please retry it?

Good luck!

**Comment by rzeldent** [comment]
I checked the code and added a message: 401 Unauthorized so it will be clearer that there is an authentication problem.
This has just been merged in the main branch PR #48

**Comment by jkykayex** [comment]
> I checked the code and added a message: 401 Unauthorized so it will be clearer that there is an authentication problem. This has just been merged in the main branch PR #48

Hey, Thanks for the replies and help - Unfortunately I failed! 

First, I changed the password here to something super easy (12345678):

<img width="370" alt="image" src="https://user-images.githubusercontent.com/67133996/227136796-6bd26995-d824-4564-9a77-3b1ab18b457a.png">


Then went on to using the web tool which failed and then I tried using VS code like you with that simple password...

<img width="1611" alt="image" src="https://user-images.githubusercontent.com/67133996/227137370-9848b3ac-013d-45c6-b6f8-85335e218127.png">


This also failed and in checking the LED brightness settings after doing this the LED was still sitting at 0: 
<img width="1007" alt="image" src="https://user-images.githubusercontent.com/67133996/227137980-2dbcc340-aa7c-4b67-96ee-3f9c2b1fdddb.png">

What would be the next step you would recommend?

I've also tried to download an app on the ipad to do the requests over the same network (IoT) to make sure nothing funky was happening on the network side. 

Happy to do further testing, just not sure what else to do... I'm not an HTTP wizz, the only thing I can think of is wireshark?
